### ISSUE TITLE
Update for data request version 01.00.26

### ISSUE THREAD

**Issue by treerink** [EC-Earth/ece2cmor3#187]
This update will be much more work than before, due to the phase we are now in: since a few weeks manual updates for Antarctic variables and "step 3" variables complicate the update now. At the same time the last shaconemo revision 256 changes have to applied with respect to the basic lists. (For the namelist related part this is already update with Shaconemo version 256).

**Comment by treerink** [comment]
We encountered erroneous priorities in the data request. The data request 01.00.25 announcement says that several priorities have been fixed. This might have also consequences for our available lists.

**Comment by treerink** [comment]
Another thing is that PAMIP has been added in the data request version 01.00.25. The CDRMIP is still not part of this data request version.

Note that the EC-Earth community has agreed on participating in PAMIP and CDRMIP.

**Comment by treerink** [comment]
After the update of drq to version 01.00.25 and the related update of the nested cmor tables, the following new messages appear for these situations:
`````scripts
./check-for-obsolete-cmor-variables-in-json-file.py 

INFO:__main__:Obsolete target found in ../resources/nemopar.json: tomint
INFO:__main__:Obsolete target found in ../resources/nemopar.json: dissicos
INFO:__main__:Obsolete target found in ../resources/nemopar.json: pdi
INFO:__main__:Obsolete target found in ../resources/nemopar.json: poc
INFO:__main__:Obsolete target found in ../resources/nemopar.json: pocos
INFO:__main__:Obsolete target found in ../resources/nemopar.json: siflsaltbot 
`````

`````scripts
./determine-missing-variables.sh CMIP CMIP 1 1

ERROR:ece2cmor3.taskloader:The dissicos variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The talkos variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The phos variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The po4os variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The dissic variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The talk variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The ph variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The po4 variable does not appear in the CMOR table file Oyr
````` 

`````scripts
./determine-missing-variables.sh CMIP,AerChemMIP,C4MIP,DCPP,HighResMIP,ISMIP6,LS3MIP,LUMIP,PAMIP,PMIP,RFMIP,ScenarioMIP,VolMIP,CORDEX,DynVar,SIMIP,VIACSAB,DAMIP CMIP 1 1

ERROR:ece2cmor3.taskloader:The pr18O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The prsn18O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The pr2h variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The prsn2h variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The pr17O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The prsn17O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The prw17O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The prw2H variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The sw18O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The sw2H variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The prw18O variable does not appear in the CMOR table file Emon
ERROR:ece2cmor3.taskloader:The dissicos variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The talkos variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The phos variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The po4os variable does not appear in the CMOR table file Omon
ERROR:ece2cmor3.taskloader:The dissic variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The dissoc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The zooc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The bacc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The detoc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The calc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The arag variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The phydiat variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The phydiaz variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The phycalc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The phypico variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The phymisc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The zmicro variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The zmeso variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The zmisc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The talk variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The ph variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The nh4 variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The po4 variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The co3 variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The co3satcalc variable does not appear in the CMOR table file Oyr
ERROR:ece2cmor3.taskloader:The co3satarag variable does not appear in the CMOR table file Oyr
`````

**Comment by aearamos** [comment]
I got the same errors when generating the nemo xml files from 
generate-ec-earth-namelists.sh and most of my file_def files came empty. I know some of the variables are identified as missing, ignored or identified missing, but shouldn't the appear in the files?
Thanks!

**Comment by treerink** [comment]
Hi Arthur,

Yes you will get the same errors for several of the ece2cmor3 scripts. We have to update for the data request update, but that is a lot of work in the current phase. Hopefully I get further on this, this week.

**Comment by treerink** [comment]
In #192 another update of these nested cmor tables for data request version 01.00.26

**Comment by treerink** [comment]
In 75db08d25bcd2821318127d461a5873f8371dda1 the basic lists for data request 01.00.26, PAMIP and Shaconemo r256 are updated. And the manual edits have reapplied. This concerns: Moving the IfxAnt and ImonAnt (did not encounter IyrAnt anylonger) variables and the CFsubhr AerChemMIP variables from the basic identified missing file to the basic ignored file, and add the Eday LS3MIP step 3 related variables to the basic identified missing file. 

Note that though all nemo non-dummy Shaconemo variables actually are available now, they have been added here to the identified missing file for completeness by using an empty nemopar.json at the moment of creating this files: so all nemo available variables are now in the identified missing file (with comprehensive comment).

**Comment by treerink** [comment]
Just to remind ourselves:

We found during debugging this issue that if in a later stage we want to get the latest data request working with the latest nested cmor tables, then we have to change in `components.py`:
`````scripts
ocnBgChem ==> ocnBgchem
`````

**Comment by treerink** [comment]
The
`````scripts
ocnBgChem ==> ocnBgchem
`````
adjustment in the data request is addressed in ad659431dce8bba629f6487cdb007ea0220c1552

**Comment by treerink** [comment]
All issues in this issue are addressed, so closing the issue.

However, note that the `cmor tables` are still at data request version 01.00.24 while the rest is updated to the current latest data request version 01.00.26, covered by #196.
